packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/ropassword/overview.md

```diff
@@ -0,0 +1,3 @@
+The Resource Owner Password flow is rarely the recommended approach. It is intended for applications for which no other flow works, as it requires your application code to be fully trusted and protected from credential-stealing attacks. It is made available primarily to provide a consistent and predictable integration pattern for legacy applications that can't otherwise be updated to a more secure flow such as Authorization Code flow. This should be your last option, not your first choice.
```

Comment:
```suggestion
The Resource Owner Password flow is rarely the recommended approach. It is intended for applications for which no other flow works, as it requires your application code to be fully trusted and protected from credential-stealing attacks. It is made available primarily to provide a consistent and predictable integration pattern for legacy applications that can't otherwise be updated to a more secure flow such as the Authorization Code flow. This should be your last option, not your first choice.
```